kartograf/util.py

```diff
@@ -51,37 +51,58 @@ def rir_from_str(maybe_rir):
     raise Exception("No RIR found in String")
 
 
-def check_compatibility():
+def get_rpki_wanted_version():
+    '''Return the rpki-client version defined in the Nix environment'''
+    rpki_path = subprocess.check_output(
+        ["which", "rpki-client"]).decode().strip()
+    try:
+        derivation = subprocess.check_output(["nix", "derivation", "show", rpki_path])
```

Comment:
yea thanks, I was thinking of Nix as our versions "source of truth" in order to avoid having to set the RPKI version as a global var, but reverted to that.